packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -182,4 +266,62 @@ contract SecurityStore is DataStore
         wallets[wallet].inheritor = who;
         wallets[wallet].lastActive = uint128(now);
     }
-}
\ No newline at end of file
+
+    function cleanRemovedGuardians(address wallet)
+        private
+    {
+        Wallet storage w = wallets[wallet];
+
+        for (uint i = 0; i < w.guardians.length; i ++) {
+            Data.Guardian memory g = w.guardians[i];
+            if (isGuardianRemoved(wallet, g.addr)) {
+                Data.Guardian memory lastGuardian = w.guardians[w.guardians.length - 1];
+
+                if (g.addr != lastGuardian.addr) {
+                    w.guardians[i] = lastGuardian;
+                    w.guardianIdx[lastGuardian.addr] = i + 1;
+                }
+                w.guardians.pop();
+                delete w.guardianIdx[g.addr];
+                delete w.guardianValidSince[g.addr];
+                delete w.guardianValidUntil[g.addr];
+            }
+        }
+    }
+
+    function isGuardianValid(address wallet, address guardian)
+        view
+        private
```

Comment:
I've got a formatting warning here that `private` should be declared before `view`. Also for the functions below this one.

Reply:
fixed